src/Tasks/Common/Resources/Strings.resx

```diff
@@ -604,4 +604,8 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1125: Publishing to a single-file is only supported for netcoreapp target.</value>
     <comment>{StrBegin="NETSDK1125: "}</comment>
   </data>
+  <data name="Crossgen2RequiresSelfContained" xml:space="preserve">
+    <value>NETSDK1126: Publishing ReadyToRun using Crossgen2 is only supported for self-contained applications for now.</value>
+    <comment>{StrBegin="NETSDK9999: "}</comment>
```

Comment:
This has the wrong version, I though I added validation for this. Is there a test failing on it.

Reply:
Yes, there is a test failing with "Expected string to start with \n"{StrBegin=\"NETSDK1126: \"}" because localization instructions should indicate invariant error code as preceding translatable message., but \n"{StrBegin=\"NETSDK9999: \"}" differs near "999" (index 17)."

Reply:
Good catch. Yes this was an oversight from a previous attempt to use 9999 as a code. Fixed